build/typings/index.d.ts

```diff
@@ -276,6 +276,50 @@ declare namespace dashjs {
 
     export namespace MediaPlayer {
         export const events: MediaPlayerEvents;
+        export const errors: MediaPlayerErrors;
+    }
+
+    interface MediaPlayerErrors {
+        MANIFEST_LOADER_PARSING_FAILURE_ERROR_CODE:          10;
+        MANIFEST_LOADER_LOADING_FAILURE_ERROR_CODE:          12;
```

Comment:
Need to update numbers here, right?

Reply:
a rebase has been done, it should be ok now.